docs/en_US/Nnictl.md

```diff
@@ -650,3 +651,37 @@ Debug mode will disable version check function in Trialkeeper.
     ```bash
     nnictl --version
     ```
+
+<a name="hdfs"></a>
+
+![](https://placehold.it/15/1589F0/000000?text=+) `Manage hdfs`
+
+* __nnictl hdfs set__
+  * Description
+
+    set the host and userName of hdfs
+
+  * Usage
+
+    ```bash
+    nnictl hdfs set [OPTIONS]
+    ```
+
+  * Options
+
+  |Name, shorthand|Required|Default|Description|
+  |------|------|------ |------|
+  |--host|  True| |The host of hdfs|
+  |--user_name|  True| |The userName of hdfs|
+
+* __nnictl hdfs clean__
+
+  * Description
+
+    Clean up all files created by nni in hdfs
```

Comment:
better to make it more clear: Clean up the code files that nni automatically copied to hdfs. This command deletes all such files under the `user_name`.

Reply:
fixed, changed to use `Clean up the code files that nni automatically copied to hdfs. This command deletes all such files under the user_name.`